src/Notebooks/FeatureEngineering.ipynb

```diff
@@ -241,34 +241,36 @@
-      "text/plain": [
-       "<matplotlib.figure.Figure at 0x7f917126c6a0>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
+     "ename": "NameError",
```

Comment:
why is there an error?

Reply:
I didn't run anything. Just looking at your pull request and seeing an error.

Reply:
Ok gotcha. I'll rerun the notebook and then resubmit the PR